lib/src/presentation/widget/card_list_builder.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:hearthstoneapp/src/core/util/api_response_state.dart';
+import 'package:hearthstoneapp/src/presentation/bloc/favorites_bloc.dart';
+import 'package:hearthstoneapp/src/presentation/widget/card_list_container.dart';
+import 'package:hearthstoneapp/src/presentation/widget/not_found.dart';
+import 'package:hearthstoneapp/src/presentation/widget/welcome.dart';
+
+class CardListBuilder extends StatelessWidget {
+  final Stream<ApiResponse> stream;
+  final FavoritesBloc favoritesBloc;
+
+  const CardListBuilder(this.stream, this.favoritesBloc);
+
+  @override
+  Widget build(BuildContext context) {
+    return StreamBuilder<ApiResponse>(
+      stream: stream,
+      builder: (context, snapshot) {
```

Comment:
trailling comma and specify context/snapshot types

Reply:
Added